okapi-core/src/test/java/org/folio/okapi/service/impl/DockerModuleHandleTest.java

```diff
@@ -88,7 +89,6 @@ public void testDockerVersionAtLocal(TestContext context) {
       "mod-users-5.0.0-SNAPSHOT", ports, "localhost", 9232, new JsonObject());
 
     dh.getUrl("/version", res -> {
-      Assume.assumeTrue(res.succeeded());
```

Comment:
If you use
* `context.asyncAssertSuccess(res -> {`
* `context.asyncAssertFailure(ex -> {`
* `context.verify(verify -> {`

you can
* avoid manual `async`
* avoid `async.await` 
* avoid `context.assertTrue(res.failed())` and `context.assertTrue(res.succeeded())`
* and use `Assume.assume...` inside an asyncAsserSuccess/asyncAssertFailure/verify block to properly skip a test so that JUnit reports it as skipped.